src/sgame/sg_bot_util.cpp

```diff
@@ -955,11 +955,18 @@ botTarget_t BotGetRetreatTarget( const gentity_t *self )
 botTarget_t BotGetRoamTarget( const gentity_t *self )
 {
 	botTarget_t target;
-	vec3_t targetPos;
+	glm::vec3 point;
 
-	BotFindRandomPoint( self->s.number, targetPos );
+	int botClientNum = self->s.number;
+	if ( !BotFindRandomPointInRadius( botClientNum, g_entities[ botClientNum ].s.origin, point, 2000 ) )
```

Comment:
Use `self` instead of going through g_entities

Reply:
done